packages/backend/src/accounting/transfers.ts

```diff
@@ -16,57 +16,90 @@ type TransfersError = {
   error: TransferError
 }
 
-export interface CreateTransferOptions {
-  id?: string | bigint
+export interface NewTransferOptions {
```

Comment:
Why "New"?

Reply:
vs `post`ing or `void`ing an existing transfer

Reply:
But why not keeping `CreateTransferOptions` or just `TransferOptions`? 

I don't have strong feelings about it but my first thought was "what is `OldTransferOptions`?" Maybe that's just me, though 😅

Reply:
I think I get it now from reading your[ comment here](https://github.com/interledger/rafiki/issues/965#issuecomment-1380590708).